src/js/App/GlobalFilter/TagsModal.js

```diff
@@ -182,6 +182,7 @@ const TagsModal = ({ isOpen, filterTagsBy, onApplyTags, toggleModal, selectedTag
       selected={[tagsSelected, sidsSelected]}
       onApply={() => onApplyTags(tagsSelected, sidsSelected)}
       title="Select one or more tags/SAP IDs (SID)"
+      ouiaId="filter-tags-modal"
```

Comment:
Assuming this modal is only used by the global filter, probably `global-filter-tags-modal` for the OUIA